cypress/e2e/inventory/search/browse-contributors-linked.cy.js

```diff
@@ -0,0 +1,129 @@
+import getRandomPostfix from '../../../support/utils/stringTools';
+import TestTypes from '../../../support/dictionary/testTypes';
+import DevTeams from '../../../support/dictionary/devTeams';
+import Permissions from '../../../support/dictionary/permissions';
+import TopMenu from '../../../support/fragments/topMenu';
+import DataImport from '../../../support/fragments/data_import/dataImport';
+import Users from '../../../support/fragments/users/users';
+import JobProfiles from '../../../support/fragments/data_import/job_profiles/jobProfiles';
+import Logs from '../../../support/fragments/data_import/logs/logs';
+import InventoryInstance from '../../../support/fragments/inventory/inventoryInstance';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import BrowseContributors from '../../../support/fragments/inventory/search/browseContributors';
+import QuickMarcEditor from '../../../support/fragments/quickMarcEditor';
+import MarcAuthority from '../../../support/fragments/marcAuthority/marcAuthority';
+import MarcAuthorities from '../../../support/fragments/marcAuthority/marcAuthorities';
+
+describe('Browse Contributors in Inventory (linked records)', () => {
```

Comment:
Maybe here better named like in testrail:
Inventory:Contributors Browse?

Reply:
Updated block name to "Inventory: Contributors Browse"